scripts/setup.py

```diff
@@ -0,0 +1,21 @@
+from data_store.const import OrganisationTypeEnum
+from data_store.db import db
+from data_store.db.entities import Organisation
+
+
+def set_organisation_type_to_local_authority():
+    """
+    Set the organisation type of all organisations in the database to LOCAL_AUTHORITY
+    """
+    local_authority = OrganisationTypeEnum.LOCAL_AUTHORITY
+    session = db.session
+
+    try:
+        session.query(Organisation).update({"organisation_type": local_authority})
+        session.commit()
+        print("Organisation types updated successfully")
+    except Exception as e:
+        session.rollback()
+        print(f"Something went wrong: {e}")
+    finally:
+        session.close()
```

Comment:
Hey hey 👋 Now that we're just adding a new column, I think we include a bit of nuance on the "no data migrations in alembic" rule 👀 

In this case I think doing an UPDATE statement to migrate _existing_ records is OK - it's just _new_ data that we shouldn't be adding.

So I think we could make the column non-nullable immediately, and then the alembic migration can:

1) Add the column as nullable=True
2) Update the rows to set the organisation_type to LA
3) Update the column to set it as nullable=False

---

This should work in a sane way, and not break anything with eg our test setup because when the migrations run there, the DB will be empty.